ios/FluentUI.Demo/FluentUI.Demo/Demos/SideBarDemoController.swift

```diff
@@ -0,0 +1,122 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import FluentUI
+import UIKit
+
+class SideBarDemoController: DemoController {
+	private var sideBar: SideBar?
+	private var contentViewController: UIViewController?
+
+	override func viewDidLoad() {
+		super.viewDidLoad()
+
+		container.addArrangedSubview(createButton(title: "Show side bar", action: #selector(presentSideBar)))
+	}
+
+	@objc private func presentSideBar() {
+		let controller = UIViewController(nibName: nil, bundle: nil)
+		contentViewController = controller
+		controller.modalPresentationStyle = .fullScreen
+		controller.view.backgroundColor = Colors.background1
```

Comment:
minor: can we avoid unnecessary references to the monolithic colors file? The goal is eventually to break that up